lib/index.js

```diff
@@ -143,6 +147,36 @@ function change (data, callback) {
 }
 exports.change = change;
 
+function checkForScopedDependencies(data, callback) {
+    if (!data.json['dist-tags']) {
```

Comment:
The dist-tags of `@foo/B` aren't relevant here, as the way this is currently set up, it calls `one` on scoped packages, which doesn't include another call to `checkForScopedDependencies`.